amplify-tools/amplify-tools-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+apply plugin: 'groovy'
+apply plugin: 'idea'
+apply plugin: 'java-gradle-plugin'
+apply plugin: 'maven-publish'
+
+apply from: rootProject.file('gradle-mvn-push.gradle')
+
+sourceSets.main.groovy.srcDirs = ["src/main/groovy"]
+
+dependencies {
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+}
+
+publishing {
```

Comment:
We have a default `publishing` rule available int `configuration/publishing.gradle`. If you import that file, at the top of this one, then you don't need a separate block, here.

The other modules use `gradle.properties` to define Maven publishing fields, and then include that central publishing script, which is a more elaborate version of this block, here.

-------

Update: oh, right, this is a separate _project_. Never-mind!

Reply:
you're not thinking fourth dimensionally